mnoptical/network.py

```diff
@@ -70,7 +70,7 @@ def add_amplifier(self, name, amplifier_type='EDFA', **params):
         return amplifier
 
     def add_link(self, src_node, dst_node, src_out_port=-1,
-                 dst_in_port=-1, boost_amp=None, spans=None):
+                 dst_in_port=-1, boost_amp=None, spans=None, **params):
```

Comment:
Probably should have been there anyway.

Reply:
But, it should be documented in the docstring.

Reply:
Got it.. Do you mean, documented as in I should add a comment here clarifying that these are span parameters? I could change the argument name of **params in Span() to **span_params.. Does that sound good?